api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -4163,6 +4181,10 @@
             "description": "storageClassName is the name of the StorageClass required by the claim. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1",
             "type": "string"
           },
+          "volumeAttributesClassName": {
+            "description": "volumeAttributesClassName may be used to set the VolumeAttributesClass used by this claim. If specified, the CSI driver will create or update the volume with the attributes defined in the corresponding VolumeAttributesClass. This has a different purpose than storageClassName, it can be changed after the claim is created. An empty string value means that no VolumeAttributesClass will be applied to the claim but it's not allowed to reset this field to empty string once it is set. If unspecified and the PersistentVolumeClaim is unbound, the default VolumeAttributesClass will be set by the persistentvolume controller if it exists. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#volumeattributesclass (Alpha) Using this field requires the VolumeAttributesClass feature gate to be enabled.",
```

Comment:
"If the VolumeAttributesClass does not exist, the ModifyVolumeStatus will be set to Pending and retry in a time interval. "?

Reply:
Sorry I suggested at the wrong place, Joe suggested to put at https://github.com/kubernetes/kubernetes/blob/36f6f501057624e8cb0f2001394f633a6516e7fe/staging/src/k8s.io/api/core/v1/types.go

Reply:
How about this? @sunnylovestiramisu 

If a PeristentVolumeClaim is created with a non-exist VolumeAttributesClass, its ModifyVolumeStatus and phase will be set to "Pending" state and retry in the next reconcile loop. But if a PersistentVolumeClaim were bound to a PersistentVolume, its phase wouldn't be affected by the VolumeAttributesClass.